admin/partials/nginx-helper-general-options.php

```diff
@@ -659,9 +710,14 @@
 					echo wp_kses(
 						sprintf(
 							'%1$s<br /><br />%2$s<strong>%3$s</strong>',
-							esc_html__( 'Can\'t write on log file.', 'nginx-helper' ), esc_html__( 'Check you have write permission on ', 'nginx-helper' ), esc_url( $log_path . 'nginx.log' )
+							esc_html__( 'Can\'t write on log file.', 'nginx-helper' ),
+							esc_html__( 'Check you have write permission on ', 'nginx-helper' ),
+							esc_url( $log_path . 'nginx.log' )
```

Comment:
This is not correct way to make small string translatable. We need to allow entire string translatable.